src/Cli/dotnet/commands/dotnet-tool/update/ToolUpdateCommand.cs

```diff
@@ -43,16 +44,17 @@ public ToolUpdateCommand(
                         localToolsResolverCache,
                         reporter);
 
+            _global = result.GetValue(ToolInstallCommandParser.GlobalOption);
+            _toolPath = result.GetValue(ToolInstallCommandParser.ToolPathOption);
+            DirectoryPath? location = string.IsNullOrWhiteSpace(_toolPath) ? null : new DirectoryPath(_toolPath);
```

Comment:
Not sure if this is the right way to handle `--tool-path` scenario. `_store` is not actually used in that case, but I think it would be more correct and future-proof to supply it with the tool path anyway.